packages/deprecation-crawler/src/lib/commands/default.command.ts

```diff
@@ -24,16 +24,26 @@ export const defaultCommand: YargsCommandObject = {
             crawl,
             addVersion,
             (config: CrawlConfig): CrawlerProcess =>
-              askToSkip('Grouping?', group(config), {
-                precondition: async (r) =>
-                  r.deprecations?.some((d) => !d.group),
-              }),
+              getInteractive()
+                ? askToSkip('Grouping?', group(config), {
+                    precondition: async (r) =>
+                      r.deprecations?.some((d) => !d.group),
+                  })
+                : group(config),
             saveDeprecations,
             (config: CrawlConfig): CrawlerProcess =>
-              askToSkip('Update Formatted Output?', format(config), {
-                precondition: async (r) => r.deprecations?.length > 0,
-              }),
-            updateRepository,
+              getInteractive()
+                ? askToSkip('Update Formatted Output?', format(config), {
+                    precondition: async (r) =>
+                      r.deprecations?.length > 0 && getInteractive(),
```

Comment:
`getInteractive()` is already check on line 35

```suggestion
                      r.deprecations?.length > 0,
```